page.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>KMarketer</title>
+    <meta charset="UTF-8">
+    <style>
+        body {
+            background-color: rgba(0, 0, 0, 0.1);
+            background-image: url("background.png");
+            background-size: cover;
+            background-attachment: fixed;
+            background-position: center;
+            color: white;
+            text-align: center;
+        }
+
+        h1,
+        h4 {
+            font-family: "Lucida Console", "Courier New", monospace;
+        }
+
+        h1 {
+            margin-top: 330px;
+            width: 100%;
+            font-size: 70px;
+        }
+
+        h4 {
+            font-size: 25px;
+
+        }
+
+        p {
+            position: fixed;
+            top: 0;
+            left: 300px;
+            font-weight: bold;
+            transform: translateX(-50%);
+            font-size: 30px;
+            margin-left: 20px;
+            margin-right: 20px;
+        }
+
+        .top-nav {
+            position: fixed;
+            top: 38px;
+            right: 275px;
+        }
+
+        .top-nav ul {
+            list-style: none;
+            margin: 0;
+            padding: 0;
+        }
+
+        .top-nav li {
+            display: inline-block;
+            margin-left: 40px;
+        }
+
+        .top-nav li:first-child {
+            margin-left: 0;
+        }
+
+        .top-nav a {
+            text-decoration: none;
+            color: white;
+            font-size: 20px;
+            font-weight: bold;
+        }
+
+        input[type="button"]{
+            margin: 10px;
+            padding: 10px 36px 10px 36px;
+            color:white;
+            height: 68px;
+            width: 205px;
+            border-radius:37px;
+            font-family: "Lucida Console", "Courier New", monospace;
+            font-size: 18px;
+            font-weight: bold;
+        }
+
+        input[value="Get Quote Now"]
+        {
+            background-color:#FFA62B;
+        }
+
+        input[value="Learn More"]
+        {
+            background-color: transparent;
+            border-color: white;
+        }
+    </style>
+</head>
+
+<body>
+
+    <h1>We Take Care of <br> the Future</h1>
+    <h4>We know how large objects will act, but things on a <br><br> small scale just do not act that way.</h4><br>
```

Comment:
double br is not the best idea. You've learned "line-height" CSS property. Use it instead of adding`br` to get more space